packages/medusa/src/api-v2/admin/price-lists/validators.ts

```diff
@@ -104,48 +105,27 @@ export class AdminPostPriceListsPriceListReq {
   rules?: Record<string, string[]>
 }
 
-export class AdminPostPriceListsPriceListPricesBatchAddReq {
-  @IsOptional()
-  @IsArray()
-  prices: AdminPriceListPricesCreateReq[]
-}
-
-export class AdminPostPriceListPriceBatchUpdate {
-  @IsOptional()
-  @IsArray()
-  prices: AdminPostPriceListPriceUpdate[]
-}
-
-export class AdminPostPriceListsPriceListPricesBatchRemoveReq {
-  @IsArray()
-  @IsString({ each: true })
-  ids: string[]
-}
-
-export class AdminPostPriceListPriceUpdate {
-  @IsString()
-  id: string
-
-  @IsString()
-  variant_id: string
-
-  @IsOptional()
-  @IsString()
-  currency_code?: string
-
-  @IsOptional()
-  @IsInt()
-  amount?: number
-
-  @IsOptional()
-  @IsInt()
-  min_quantity?: number
-
-  @IsOptional()
-  @IsInt()
-  max_quantity?: number
-
-  @IsOptional()
-  @IsObject()
-  rules?: Record<string, string>
-}
+export const AdminCreatePriceListPrice = z.object({
```

Comment:
Also define the type of it like this above it:
```suggestion
export type AdminCreatePriceListPriceType = z.infer<
  typeof AdminCreatePriceListPrice
>
export const AdminCreatePriceListPrice = z.object({
```

And then pass that to `AuthenticatedMedusaRequest<AdminCreatePriceListPriceType>`

This applies for pretty much all validators